tests/experimental_combination/combine_wzdx_test.py

```diff
@@ -2,7 +2,7 @@
 
 from shapely.geometry import Point
```

Comment:
Should we rename this for "icone_cotrip_test" to correspond with the file we're testing?

Reply:
There's also some in-code json expected object declarations that we could abstract to make it more readable. We could do that in a separate PR though.

Reply:
That file is no longer used, I went ahead and removed it, along with the test